salt/modules/cmdmod.py

```diff
@@ -1156,6 +1192,20 @@ def run(
 
       .. versionadded:: 2019.2.0
 
+    :param list success_stdout: This parameter will be allow a list of
+        strings that when found in standard out should be considered a success.
+        If stdout returned from the run matches any in the provided list,
+        the return code will be overridden with zero.
+
+      .. versionadded:: Silicon
```

Comment:
Can you just add the version number. Since we do not use date based versioning this can just be 3004

Reply:
Done